packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -29,15 +30,55 @@ export const TerminalConfigSchema: PreferenceSchema = {
             type: 'boolean',
             description: 'Enable ctrl-v (cmd-v on macOS) to paste from clipboard',
             default: true
-        }
+        },
+        'terminal.integrated.fontFamily': {
+            type: 'string',
+            description: 'Controls the font family of the terminal, this defaults to `#editor.fontFamily#`\'s value.',
+            default: EDITOR_FONT_DEFAULTS.fontFamily
+        },
+        'terminal.integrated.fontSize': {
+            type: 'number',
+            description: 'Controls the font size in pixels of the terminal.',
+            default: EDITOR_FONT_DEFAULTS.fontSize
+        },
+        'terminal.integrated.fontWeight': {
+            type: 'string',
+            enum: ['normal', 'bold', '100', '200', '300', '400', '500', '600', '700', '800', '900'],
+            description: 'The font weight to use within the terminal for non-bold text.',
+            default: 'normal'
+        },
+        'terminal.integrated.fontWeightBold': {
+            type: 'string',
+            enum: ['normal', 'bold', '100', '200', '300', '400', '500', '600', '700', '800', '900'],
+            description: 'The font weight to use within the terminal for bold text.',
+            default: 'bold'
+        },
+        'terminal.integrated.letterSpacing': {
+            description: 'Controls the letter spacing of the terminal, this is an integer value which represents the amount of additional pixels to add between characters.',
+            type: 'number',
+            default: 1
```

Comment:
should be `0`, cf. editor settings

Reply:
It is intentionally 1 as I found 0 too narrow.